app/controllers/experiments_controller.rb

```diff
@@ -535,6 +535,10 @@ def experiment_moes
     render json: moes_info
   end
 
+  def experiment_results
```

Comment:
results_info? Po pierwsze to krok w kierunku pozbycia się prefiksu, po drugie 'info' sugeruje więcej albo inne informacje.